### DISCUSSION TITLE
Denied access

### DISCUSSION THREAD

**Discussion by Mrnovey** [community/community#138658] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have an account on Bitcloud but unable to link and access my account. My address is eme2da@gmail. I am unable to go to my account. Where do I go to link to Bitcloud and see my account?

**Comment by ebndev** [comment]
Hi @Mrnovey,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try contacting Bitcloud Support. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
